src/main/java/com/example/library/AppConfig.java

```diff
@@ -1,9 +1,14 @@
 package com.example.library;
 
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ComponentScan({""})
 public class AppConfig {
+    @Bean
+    public ModelMapper modelMapper() {
```

Comment:
Dla prostego mapowania jak najbardziej ok.
Jako alternatywa (my w niektórych miejscach używamy) mapstruct ale to tylko jako ifno